pkg/roachprod/install/cluster_synced.go

```diff
@@ -48,6 +48,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const SIGKILL = 9
```

Comment:
Nit: The above package "unix" has a `SIGKILL` as well, we use `SIGHUP` down the line with it for the certs. But it requires an `int` recast, which is a bit annoying.

Reply:
Good point, updated here and in the other places you mentioned.